docs/using.adoc

```diff
@@ -100,11 +102,13 @@ the version of the image to be used.
 .  Start ADB using `vagrant up`. You will find detailed installation
 instructions in the link:docs/installing.adoc[Installation document].
 
-.  Configure the environment and download the required TLS certificates
+=== Configure your environment
```

Comment:
+1, chunking the 3 sections below makes them quickly accessible to the user through the TOC itself.